index.html

```diff
@@ -19,7 +19,7 @@
   <meta name="msapplication-TileColor" content="#da532c">
   <meta name="msapplication-config" content="/favicon/browserconfig.xml?v=2.0.22">
   <meta name="theme-color" content="#ffffff">
-  <meta property="og:image" content="/active/preview.png?{{currentQueries}}">
+  <meta property="og:image" content="/active/preview.png?{{.CurrentQueries}}">
```

Comment:
I see that main idea is to pass CurrentQueries here.
but can't find where is this file located? (active/preview.png)

Reply:
it address of image generator backend